packages/cursorless-vscode-e2e/src/suite/serializeHeader.ts

```diff
@@ -0,0 +1,53 @@
+import { Range } from "@cursorless/common";
+
+interface SerializeHeaderArg {
+  prefix?: string;
+  header: string | undefined;
+  scopeIndex: number | undefined;
+  targetIndex: number | undefined;
+  range?: Range;
+}
+
+/**
+ * Constructs a header string from a configuration object. For example:
+ *
+ * ```
+ * serializeHeader({
+ *  prefix: "Leading delimiter",
+ *  header: "Content",
+ *  scopeIndex: 1,
+ *  targetIndex: 2,
+ *  range: new Range(new Position(1, 2), new Position(3, 4)),
+ * }) === "[#1 Leading delimiter: Content 2] =1:2-3:4"
```

Comment:
Did I get this right? If so, it feels a bit awkward to have the target index be all the way at the end; feels like it's a higher part of the hierarchy than the header and prefix. I'd argue something like

```
[#1.2 Leading delimiter: Content]
```

Reply:
Sure. Shall I make that change then?

Reply:
fixed